### DISCUSSION TITLE
duplicate args are removed

### DISCUSSION THREAD

**Discussion by nnaydenow** [storybookjs/storybook#24558] [Help]
### Describe the bug

I have a custom-elements.json that describes my custom elements. I use this file to display documentation for my custom elements in Storybook.

The issue I'm facing is that some of my components have CSS part and a slot with the same name, but in Storybook, only one of them is shown. Here's how my custom element is described in "custom-elements.json":

```json
{
  "cssParts": [
    {
      "description": "Some header CSS part",
      "name": "header"
    }
  ],
  "slots": [
    {
      "name": "header",
      "description": "Some header slot"
    }
  ]
}
```
How can I make sure that both the CSS part and the slot are displayed in the args table in Storybook?

Please provide a solution for this.

**Comment by Lutterc** [comment]
Hello, this problem has been solved

**Reply by valentinpalkovic** [reply]
Hi @Lutterc 

Can you provide a solution to @nnaydenow if you think the issue has been solved?

**Comment by robinsummerhill** [comment]
I am still seeing this issue with Storybook 8.3.2 using @storybook/web-components and @storybook/web-components-vite:

custom-elements.json extract:
```
 "cssParts": [
    {
      "description": "The container that wraps the avatar's icon.",
      "name": "icon"
     },
  ],
  "slots": [
    {
      "description": "The default icon to use when no image or initials are present. Works best with `<flow-ui-icon>`.",
      "name": "icon"
     }
  ]
```

`icon` is only appearing in the CSS Parts table. If I change the slot name to `iconSlot` then it appears in the the Slot table too.

The same thing happens if there is duplication between any property, slot, or CSS part.
